Core/src/main/java/com/plotsquared/core/command/Visit.java

```diff
@@ -235,8 +237,8 @@ private void visit(
                         } else {
                             final UUID uuid = uuids.toArray(new UUID[0])[0];
                             PlotQuery query = PlotQuery.newQuery();
-                            if (Settings.Teleport.VISIT_MERGED_OWNERS) {
```

Comment:
We use that format everywhere, though it's redundant but makes reading easier.

Reply:
Was just to make the conditional operators shorter in the if else statements tbh